src/lib/utils/format.ts

```diff
@@ -0,0 +1,13 @@
+function createListFormatter() {
+  if (Intl.ListFormat) {
+    return new Intl.ListFormat("en", { style: "long", type: "conjunction" });
+  }
+
+  return {
+    format(items: any[]) {
+      items.join(",");
```

Comment:
just a nit, but maybe this should be `, ` (with a space after the comma in the join)